Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN make build
 
 FROM registry.svc.ci.openshift.org/ocp/4.6:cli as origincli
 
-FROM registry.svc.ci.openshift.org/ocp/4.6:base-8
+FROM registry.svc.ci.openshift.org/ocp/4.6:base
```

Comment:
@jcantrill Isn't this change going to break again our local builds, as fixed by 799647bed5d355a19240c6b11eef87e2ffd56f74

Reply:
What @periklis said. If local builds are OK then I'm happy.